src/main/java/nextstep/auth/authentication/TokenAuthenticationConverter.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.auth.authentication;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServletRequest;
+import nextstep.auth.token.TokenRequest;
+
+public class TokenAuthenticationConverter implements AuthenticationConverter {
+
+    @Override
+    public AuthenticationToken convert(HttpServletRequest request) {
+        // TODO: request에서 AuthenticationToken 객체 생성하기
+        try {
+            String requestData = request.getReader()
```

Comment:
```suggestion
            String request = request.getReader()
```

Data라는 단어가 구체적으로 무엇을 표현하는지 알기 어려우니 빼는게 좋겠네요.